move-boll/main.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
good work!
But if we move our field to the centre of the screen, the code breaks down